### ISSUE TITLE
Citibus 61R is missing

### ISSUE THREAD

**Issue by mulderbert** [alvinhkh/buseta#21]
88R (都係居民巴士）can be found but 61R is missing

**Comment by winstonma** [comment]
You are right, I could find 61R in [Citybus Route in Open Data](https://rt.data.gov.hk/v1.1/transport/citybus-nwfb/route/ctb).

AFAIK the route info is collected from [NWST Site](https://github.com/alvinhkh/buseta/blob/master/app/src/main/java/com/alvinhkh/buseta/nwst/NwstService.kt). So not sure if there are some difference between the datasets.

**Comment by mulderbert** [comment]
Further update: I found that 88R cannot be searchable also. But that route since I had saved before, the ETA info can still be seen in my list but you just can't find such route using search right now… :-(

**Comment by noname617** [comment]
以上兩條，鑑於ctb問題，用政府果條嘗試用NR61,NR88，一會都會出，之前點解可以直接搵到，就係ctb，分類用61r,88r黎出結果
